tools/sigh.js

```diff
@@ -273,13 +274,17 @@ function test(args) {
     let runner = `
       import mocha from '${mochaInstanceFile}';
       ${chainImports.join('\n    ')}
-      mocha
-        .grep(${JSON.stringify(options.grep || '')})
-        .run(function(failures) {
-          process.on("exit", function() {
-            process.exit(failures > 0 ? 1 : 0);
+      import devtoolsChannelProvider from '${fixPathForWindows(path.resolve(__dirname, '../runtime/debug/devtools-channel-provider.js'))}';
+      (async () => {
+        ${options.explore ? 'console.log("Waiting for Arcs Explorer"); await devtoolsChannelProvider.get().ready;' : ''}
```

Comment:
A way to simplify this might be to move this logic (the import, logging and get) to another module, and add that to the front of `chainImports` when the option is set. But up to you.

Reply:
Restructured the code a little bit as suggested, it's nicer know.